modules/select/js/select_directive.js

```diff
@@ -1276,8 +1276,11 @@
          * @param {Event}  evt        The scroll event.
          * @param {string} dropdownId The id of the dropdown that scrolled to the end.
          */
-        $scope.$on('lx-dropdown__scroll-end', async (evt, dropdownId) => {
-            if (angular.isUndefined(lxSelect.infiniteScroll) || dropdownId !== 'dropdown-' + lxSelect.uuid) {
+        $scope.$on('lx-dropdown__scroll-end', function(evt, dropdownId) {
+            if (angular.isUndefined(lxSelect.infiniteScroll) 
+                || dropdownId !== 'dropdown-' + lxSelect.uuid 
+                || !!lxSelect.loading
```

Comment:
You should not coerce `lxSelect.loading` type as `boolean` with double negation, it could be misleading
We prefer the use of the `Boolean` constructor if needed.

You can refer to the https://eslint.org/docs/rules/no-extra-boolean-cast eslnt rule which we use internally